pkg/helpers/helpers.go

```diff
@@ -0,0 +1,12 @@
+package helpers
```

Comment:
Do we really need this package? I'm afraid it might easily become one of those pools for miscellaneous stuff.

In fact, I think we already moved this func once from `common` to being internal of `main.go` for that very reason (see #226), and also because we don't want to send a message that environment variables should be freely and easily used from anywhere in the code.

Reply:
Also, a good practice if we want to unit test everything, and isolate concerns.